src/App.js

```diff
@@ -1,6 +1,7 @@
 import Layout from "./layouts/Layout";
 import "./App.css";
 import { BrowserRouter as Router } from "react-router-dom";
+require("dotenv").config();
```

Comment:
what for is that package? I understand what it should do, but .env is ignored so I don't know what is there :)

Reply:
It's for hiding url to the server to prevent access because repo is public. More info in our Slack conversation.